File: A2Z_sheet/Array/Easy/09_find_the_union/solution.py
class Solution:
    def findUnion(self,arr1,arr2,n,m):
        set_union = set()
        for num in arr1:
            set_union.add(num)
        for num in arr2:
            set_union.add(num)
        union_list = sorted(set_union)
        return union_list

File: A2Z_sheet/Array/Easy/09_find_the_union/solution.js
class Solution {
    findUnion(arr1, arr2, n, m){
        let set = new Set();
        arr1.forEach(num => set.add(num));
        arr2.forEach(num => set.add(num));
        let unionArray = Array.from(set);
        unionArray.sort((a, b) => a - b);
        return unionArray;
    }
}
